src/zc/table/fieldcolumn.py

```diff
@@ -127,7 +131,7 @@ def set(self, item, value, formatter):
     def getFieldContext(self, item, formatter):
         return None
 
-    ###### main API: input, update, and custom renderCell ######
+    # main API: input, update, and custom renderCell ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..